api/lagfest-signup/function_app.py

```diff
@@ -12,8 +12,9 @@
 
 TZ = pytz.timezone("Europe/Helsinki")
 signup_start = TZ.localize(datetime(2024, 2, 22, 16))
+signup_start_tatiseta = TZ.localize(datetime(2024, 9, 19, 16))
```

Comment:
Eikö pitäisi olla  15.8. kun lipunmyyti alkaa?

Reply:
true, and this is why we have reviewers.

Reply:
fixed (hopefully)